hardhat.config.ts

```diff
@@ -390,6 +400,12 @@ const config: HardhatUserConfig = {
       mantaMainnet: "PLACEHOLDER_STRING",
       mantaTestnet: "PLACEHOLDER_STRING",
       arbitrumSepolia: process.env.ARBITRUM_API_KEY || "",
+      // To Be Added
+      // scroll testnet
+      // scroll mainnet
+      // blast testnet
+      // zeta testnet
+      // capx testnet
```

Comment:
let's add these explorer api key configs, at least as environment variables

Reply:
ok added for scroll and blast testnet for now as those two only are part of this showcase PR.